README.md

```diff
@@ -46,10 +46,15 @@ We try our best to adhere to the set of rules and guidelines that we've come up
 We use npm and Nx to manage our workspace and dependencies. The Nx CLI offers powerful tools for development, and we recommend you install it when contributing to the project.
 
 1. Install npm and nodejs for your operating system.
-1. Install the Nx CLI:
+2. Install the Nx CLI:
    - `npm install -g nx`
-1. Install dependencies:
+3. Install dependencies:
```

Comment:
No need to number them manually, Github's Markdown will automatically number them correctly. Please put it back to this:
```md
1. Install the Nx CLI:
   - `npm install -g nx`
1. Install dependencies:
```

Also, I noticed that numbers are being incremented manually in other parts of this readme. If you have the time feel free to put them all back to 1.

Reply:
done